build_config/lib/src/key_normalization.dart

```diff
@@ -2,13 +2,25 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-String normalizeBuilderKey(String builderKey, String packageName) {
+String normalizeBuilderKeyDefinition(String builderKey, String packageName) {
+  if (builderKey.startsWith('|')) return '$packageName$builderKey';
+  if (!builderKey.contains('|')) return '$packageName|$builderKey';
+  return builderKey;
+}
+
+String normalizeBuilderKeyUsage(String builderKey, String packageName) {
   if (builderKey.startsWith('|')) return '$packageName$builderKey';
   if (!builderKey.contains('|')) return '$builderKey|$builderKey';
   return builderKey;
 }
 
-String normalizeTargetKey(String targetKey, String packageName) {
+String normalizeTargetKeyDefinition(String targetKey, String packageName) {
+  if (targetKey.startsWith(':')) return '$packageName$targetKey';
+  if (!targetKey.contains(':')) return '$packageName:$targetKey';
+  return targetKey;
+}
+
+String normalizeTargetKeyUsage(String targetKey, String packageName) {
   if (targetKey.startsWith(':')) return '$packageName$targetKey';
   if (!targetKey.contains(':')) return '$targetKey:$targetKey';
```

Comment:
specifically, why is this one `$targetKey:$targetKey` and not `$packageKey:$targetKey`?